routes/web.php

```diff
@@ -3,13 +3,14 @@
 use App\Http\Controllers\ChirpController;
 use App\Http\Controllers\CommentController;
 use App\Http\Controllers\ProfileController;
+use App\Http\Controllers\UserController;
 use Illuminate\Support\Facades\Route;
 
 Route::get('/', function () {
-    return view('welcome');
-});
+    return view('guest');
+})->middleware(['guest'])->name('guest');
```

Comment:
you shouldn't need this route. if your authenticate middleware properly redirects to the login or register page when user not authenticated this isn't necessary